### DISCUSSION TITLE
中文封面上第二导师文本超出下划线范围

### DISCUSSION THREAD

**Discussion by YmHAHAHa** [nju-lug/NJUThesis#214] [Q&A] [answered]
![image](https://github.com/nju-lug/NJUThesis/assets/42953643/3aa41503-910f-4e59-90f9-6f9e69ef3f90)

请问封面页下划线长度如何调整，目前第二导师文本超出下划线范围，模板版本1.2.1，编译环境Texlive2022

**Comment by atxy-blip** [comment, marked as answer]
有没有可能：
https://github.com/nju-lug/NJUThesis/blob/3f74337a889e11ff650aa7b3dcd1d378c245298b/docs/njuthesis-setup.def#L256

330pt 可能太长，不妨自行找一个合适的数值。

**Reply by YmHAHAHa** [reply]
感谢！
